pkg/cmd/server/api/v1/conversions.go

```diff
@@ -131,6 +131,10 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 			if len(obj.BindNetwork) == 0 {
 				obj.BindNetwork = "tcp4"
 			}
+			if obj.AllowRecursiveQueries == nil {
```

Comment:
It's on by default for 1.2/3.2 installs when NetworkManager is available. Apparently Centos cloud images don't include NM by default so we have that gap. I'll be working on closing that next week.